dev/packages/example/nginx-1.2.0/docs/README.md

```diff
@@ -1,16 +1,104 @@
+The Nginx stubstatus metrics were tested with Nginx 1.9 and are expected to work with all version >= 1.9. The logs were tested with version 1.10. On Windows, the module was tested with Nginx installed from the Chocolatey repository.
 
-### Inputs
 
-Inputs have to be copied here directly.
+## Logs
 
-## Dashboard
 
-The nginx integration comes with a predefined dashboard. 
+**Timezone support**
+
+This datasource parses logs that don’t contain timezone information. For these logs, the Elastic Agent reads the local timezone and uses it when parsing to convert the timestamp to UTC. The timezone to be used for parsing is included in the event in the event.timezone field.
+
+To disable this conversion, the event.timezone field can be removed with the drop_fields processor.
+
+If logs are originated from systems or applications with a different timezone to the local one, the event.timezone field can be overwritten with the original timezone using the add_fields processor.
+
+### Access Logs
+
+Access logs collects the nginx access logs.
+
+**Exported fields**
+
+| Field                       	| Description                                                                                                                                                                                            	| Type 	|
+|-----------------------------	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|------	|
+| nginx.access.remote_ip_list 	| An array of remote IP addresses. It is a list because it is common to include, besides the client IP address, IP addresses from headers like X-Forwarded-For. Real source IP is restored to source.ip. 	| ip   	|
+| nginx.stubstatus.active     	| The current number of active client connections including Waiting connections.                                                                                                                         	| long 	|
+| nginx.stubstatus.accepts    	| The total number of accepted client connections.                                                                                                                                                       	| long 	|
+| nginx.stubstatus.handled    	| The total number of handled client connections.                                                                                                                                                        	| long 	|
+| nginx.stubstatus.dropped    	| The total number of dropped client connections.                                                                                                                                                        	| long 	|
+
+### Error Logs
+
+Error logs collects the nginx error logs.
+
+**Exported fields**
+
+| Field                     	| Description                                                                    	| Type 	|
```

Comment:
@jen-huang @hbharding Github in markdown supports this nice collapsible markdown. Is there a way we could use this too inside Kibana? Reason is that otherwise the page could become pretty long and I'm not sure we should show all the fields by default (there can be 50-100 fields).

<details><summary>CLICK ME</summary>
<p>

#### yes, even hidden code blocks!

```python
print("hello world!")
```

</p>
</details>